libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/RustClientSessionDelegate.kt

```diff
@@ -101,7 +109,7 @@ class RustClientSessionDelegate(
                 } else {
                     clientLog.d("No session data found.")
                 }
-                client?.logout(userInitiated = false, ignoreSdkError = true)
+                client.get()?.logout(userInitiated = false, ignoreSdkError = true)
```

Comment:
This is not new, but I am wondering why we are not using `currentClient` here (instead of `client.get()?`).

Reply:
No reason, it's better to use `currentClient` indeed!